src/components/posts/AddPostForm.js

```diff
@@ -0,0 +1,58 @@
+import { useRef } from "react";
+import classes from "./AddPostForm.module.css";
+
+function AddPostForm(props) {
+  const authorInputRef = useRef();
+  // const dateInputRef = useRef();
```

Comment:
If this isn't used anymore it is likely worth deleting instead of commenting out

Reply:
see above